src/main/java/frc/robot/subsystems/arm/commands/SetArmsAngles.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+
+public class SetArmsAngles extends CommandBase {
+    private final Arm arm;
+    private final Translation2d position;
+
+    public SetArmsAngles(Arm arm, Translation2d position) {
+        this.arm = arm;
+        this.position = position;
+        addRequirements();
+    }
+
+    @Override
+    public void execute() {
+        arm.setAngle(position);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        arm.setShoulderJointPower(0);
+        arm.setElbowJointPower(0);
```

Comment:
Why is that needed?

Reply:
To avoid damaging the motors

Reply:
When I think about it the end method shouldn't really matter because it will just return to the default command (xbox/joystick control)

Reply:
When running the Joystick or Xbox control commands, if the controller isn't touched, the arm will drop (the code will essentially be `motor.set(TalonFXControlMode.PercentOutput, 0)`). Is this the desired behaviour?

Reply:
Also, dropping a link here to my comment about reducing the Joystick or Xbox "manual" control commands into one command: https://github.com/Galaxia5987/robot-2023/pull/12#discussion_r1084659696.